src/accelerate/accelerator.py

```diff
@@ -380,6 +381,8 @@ def __init__(
                 self.scaler = ShardedGradScaler(**kwargs)
             else:
                 self.scaler = torch.cuda.amp.GradScaler(**kwargs)
+                if is_xpu_available():
+                    self.scaler = torch.xpu.amp.GradScaler(**kwargs)
```

Comment:
```suggestion
                if is_xpu_available():
                    self.scaler = torch.xpu.amp.GradScaler(**kwargs)
                else:
                    self.scaler = torch.cuda.amp.GradScaler(**kwargs)
```

Reply:
sorry but there is no this api "torch.xpu.amp.GradScaler"